public/javascripts/running.js

```diff
@@ -168,39 +148,34 @@ function enableInteractive() {
 }
 
 function disableInteractive() {
-    window.hide("stop_button");
+    $("#stop_button").hide();
     sendCommand("disable_interactive_mode");
 }
 
 function sendCommand(command) {
-    var wid = window.testStatus.workerid;
+    var wid = testStatus.workerid;
     if (wid == null) return false;
-    new Ajax.Request("/api/v1/workers/" + wid + "/commands", {
-        method: "post",
-        parameters: { command: command },
-        onSuccess: function(resp) {
-            window.setTimeout("updateStatus()", 0);
-        }});
+    $.ajax({url: "/api/v1/workers/" + wid + "/commands",
+	    type: 'POST',
+            data: { command: command },
+            success: function(resp) {
+		setTimeout("update_status()", 0);
+            }});
 }
 
-function updateStatus() {
-    new Ajax.Request("/tests/" + window.testStatus.testname + "/status", {
-        method: "get",
-        dataType: 'json',
-        onSuccess: function(resp) {
-            var status = resp.responseJSON;
-            window.updateTestStatus(status);
-            window.setTimeout("updateStatus()", 3000);
-        },
-        on404: function(resp) { // no status? probably not running anymore
-          window.setTimeout(function() {location.reload();}, 1000);
-        }
-    });
+function update_status() {
```

Comment:
Actually in javascript camelCasing is the standard. So if we are going to unify (and this PR does not unify, only changes some random names), we should do it the other way around.